QQE-1290.md

```diff
@@ -0,0 +1,19 @@
+# QQE-1290 Test RestClientReactive base URL override
+
+JIRA link: https://issues.redhat.com/browse/QQE-1290
+
+Test plan covers reactive rest client's ability to override base URL.
+
+## Scope of the testing
+Verify that quarkus-reactive-rest-client is able to override base URL by annotation-based parameters on runtime.
+Tests will be executed on baremetal JVM and native.
```

Comment:
Why not Openshift?

Reply:
Technically we probably can run this on OCP but IMHO makes no sense, since only thing the tests are actually testing is overriding some config inside rest-client - which is independent of runtime env. So IMHO running this on OCP is waste of execution time.